src/commands/commandgraphicguess.cpp

```diff
@@ -0,0 +1,93 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "commandgraphicguess.h"
+#include "commandlineparser.h"
+#include "leakdetector.h"
+#include "logger.h"
+
+#include <QDir>
+#include <QFile>
+#include <QGuiApplication>
+#include <QQmlApplicationEngine>
+#include <QStandardPaths>
+#include <QTextStream>
+
+namespace {
+Logger logger(LOG_MAIN, "CommandGraphicGuess");
+
+uint32_t s_vertexShaderErrorCount = 0;
+void messageQTHandler(QtMsgType, const QMessageLogContext&,
+                      const QString& message) {
+  if (message.contains("0x80070057")) {
+    ++s_vertexShaderErrorCount;
+  }
+}
+
+}  // namespace
+
+CommandGraphicGuess::CommandGraphicGuess(QObject* parent)
+    : Command(parent, "gg", "Guess the graphic configuration") {
```

Comment:
I don't particularly like the name `gg` for this test. I would recommend changing it to something more like `gpucheck` which I feel has a stronger association with running a test to get a result.

Also, "gg" is old internet slang for good game - something you would say to someone when the game is over. And I hope user's aren't over with our product :)